components/scream/tests/scream_homme_dyn_p3/CMakeLists.txt

```diff
@@ -14,30 +14,43 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/input.yaml
                ${CMAKE_CURRENT_BINARY_DIR}/input.yaml COPYONLY)
 
 # Set homme's test options, so that we can configure the namelist correctly
+
+# Discretization/algorithm settings
 set (HOMME_TEST_NE 2)
-set (HOMME_TEST_NDAYS 1)
 set (HOMME_TEST_LIM 9)
 set (HOMME_TEST_QSIZE 6)
 set (HOMME_TEST_RSPLIT 3)
-set (HOMME_TEST_MOISTURE dry)
+set (HOMME_TEST_NDAYS 1)
+set (HOMME_TEST_TIME_STEP 600)
+SET (HOMME_THETA_FORM 1)
```

Comment:
not important, but it looks like up above you specifically switched upper-case "SET" to lower-case "set" as a change and here you are switching between upper and lower case in this list of options.  Is there any difference in cmake?

Reply:
same for configure_file below...

Reply:
Ah, yes, thanks. HOMME uses mostly upper case, while scream seems to use only lower case (which I like better, btw). Copying and pasting from HOMME left some unchanged lines.